packages/lib/event-types/getEventTypesByViewer.ts

```diff
@@ -30,6 +30,7 @@ type User = {
 type Filters = {
   teamIds?: number[];
   upIds?: string[];
+  schedulingType?: SchedulingType;
```

Comment:
What if we want to fetch round robin and collective? I think `schedulingType` should be an array.